INSTALLATION.md

```diff
@@ -61,4 +61,40 @@ TODO
 
 ### FreeBSD
 
-TODO
+Installation on FreeBSD currently requries a source build because a VAST port
+does not exist (yet).
+
+#### Service Management
+
+FreeBSD uses the [rc(8)][rc] system of startup scripts during system
+initialization and for managing services. VAST ships with an rc script (similar
+to the ones in `/etc/rc.d` and `/usr/local/etc/rc.d`) that allows
+[service(8)][service] to manage a `vast` process with the `start`, `stop`, and
+`restart` options. The rc script uses [daemon(8)][daemon] to manage the `vast`
+process.
+
+The following steps install VAST as a persistent service:
+
+1. Copy the rc script to `/usr/local/etc/rc.d`.
+
+   **Note**: this happens automatically when executing the `install` target.
+
+2. Add the following line to `/etc/rc.conf`:
```

Comment:
```suggestion
2. Add the following line to `/etc/rc.conf` to start VAST on each boot:
```

Reply:
I don't think this is necessary to add here, because the instructions are for adding VAST as a "persistent service".

Reply:
I don't think this is necessary to add here, because the instructions are for adding VAST as a "persistent service".

Reply:
This is fine then.